assets/js/mbta.ts

```diff
@@ -45,6 +45,10 @@ const branchConfig: { [key: string]: { id: string; name: string }[] } = {
       id: 'green_trunk',
       name: 'Green Line Trunk',
     },
+    {
+      id: 'green_b',
+      name: 'Green - B',
+    },
```

Comment:
question: how are stops associated with this in the bulk editor?

Reply:
I believe the association is actually in [realtime_signs/priv/signs.json](https://github.com/mbta/realtime_signs/blob/94269ffdeeb7aae269c3bcbf66c25c34b9dfaef6/priv/signs.json). As of now it looks like `green_b` doesn't exist, perhaps that needs to be added to the `realtime_signs` component of this task as well.